### ISSUE TITLE
Switch to PHP 8

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/drupal-container#32]
As the title says.

**Comment by geerlingguy** [comment]
ARM64 image is switched. Tested and working on `jeffgeerling-com`. Other image versions will be updated some time after #33 is fixed.

**Comment by geerlingguy** [comment]
Huh.

[Drupal 8/9 are not yet compatible with PHP 8](https://www.drupal.org/project/drupal/issues/3156651). Dang.

**Comment by stephenpurkiss** [comment]
Hi! I thought I would play with my Raspberry Pi 4 8Gb again - I'd used drupal-pi back in Feb/Mar this year and it worked great but I thought I'd re-install so I get a clean version now it's saying the PHP version is too old.

I know I should really probably set up everything manually but you seem to know what you're doing a lot more than my bashing the keyboard and I landed up here after trying to do "docker exec" and finding out that's not the way this stuff works so thought I'd see if it was now time to un-postpone, or if possible point me in the way of how I can do it because I'm totally lost as per usual lol!

TIA!

![Image 02-12-2022 at 08 05](https://user-images.githubusercontent.com/6438292/205246067-230339ab-6275-4af4-95fc-bcbafbd120e6.jpeg)

**Comment by geerlingguy** [comment]
I did just upgrade my php-apache-container to 8.1, so this shouldn't be too difficult: https://github.com/geerlingguy/php-apache-container

**Comment by geerlingguy** [comment]
Done!

**Comment by stephenpurkiss** [comment]
Thanks!
